job-dsl-core/src/test/groovy/javaposse/jobdsl/dsl/helpers/ScmContextSpec.groovy

```diff
@@ -1993,6 +1993,73 @@ class ScmContextSpec extends Specification {
         (1.._) * mockJobManagement.requirePlugin('perforce')
     }
 
+    def 'call perforcep4 with manual workspace'() {
+        when:
+        context.perforcep4('p4-user-creds') {
+            manual('ws', '//depot/Tools/build/...\n//ws/webapplications/helloworld/...')
```

Comment:
The view specification here looks odd. It should be something like:
{code}
            manual('ws', '//depot/Tools/build/...//ws/build/...\n//depot/webapplications/helloworld/... //ws/build/...')
{code}